use_case_examples/deployment/breast_cancer_builtin/Makefile

```diff
@@ -1,17 +0,0 @@
-# Useful for jupyter notebooks
```

Comment:
why remove these? they use the fhemodelserver stuff to build a docker, they don't need CML dependencies to boto or fast api. The test is just to export the model and build the docker which seems fine to me

Reply:
I can add them back and only test the docker build maybe yes.

Reply:
Added them back